test/morphometrics/test_compute_morphometrics.py

```diff
@@ -147,6 +209,224 @@ def test_get_axon_morphometrics_with_myelin_mask_simulated_axons(self):
             assert stats_array[ii]['gratio'] == pytest.approx(gratio_sim[ii], rel=0.1)
             assert stats_array[ii]['axon_diam'] == pytest.approx(axon_diam_sim[ii], rel=0.1)
             assert stats_array[ii]['myelin_thickness'] == pytest.approx(myelin_thickness_sim[ii], rel=0.1)
+    
+    @pytest.mark.unit
+    def test_get_axon_morphometrics_with_myelin_mask_simulated_axons_with_axon_as_ellipse(self):
+        # Simulated image of axon and myelin to test morphoterics generated by ADS when axon shape is ellipse
+        
+        self.image_sim_ellipse.generate_axon(axon_radius=50, center=[80, 80], gratio=0.9, plane_angle=0)
+        self.image_sim_ellipse.generate_axon(axon_radius=45, center=[180, 180], gratio=0.8, plane_angle=10)
+        self.image_sim_ellipse.generate_axon(axon_radius=40, center=[280, 280], gratio=0.7, plane_angle=20)
+        self.image_sim_ellipse.generate_axon(axon_radius=35, center=[380, 380], gratio=0.6, plane_angle=30)
+        self.image_sim_ellipse.generate_axon(axon_radius=30, center=[480, 480], gratio=0.5, plane_angle=40)
+        self.image_sim_ellipse.generate_axon(axon_radius=23, center=[580, 580], gratio=0.4, plane_angle=50)
+        self.image_sim_ellipse.generate_axon(axon_radius=18, center=[700, 700], gratio=0.3, plane_angle=60)
+        self.image_sim_ellipse.generate_axon(axon_radius=12, center=[800, 850], gratio=0.2, plane_angle=70)
```

Comment:
@vs74  I think here `axon_radius` should be 10 and not 12, as `axon_diam_sim` below is 20 and not 24.

Reply:
Yeah, you are right. It should be 10 instead of 12. Thanks for catching it.

Reply:
I see you changed the axon_diam_sim to 24 instead of the axon radius here to 10